### PR TITLE
PD-4602 Pilot documentation files

### PR DESCRIPTION
**Pull request by terado**

### CODE REVIEW

**Review by forjayssake — APPROVED** [review]
Understand some of this is WIP/Subject to change

File: build/automation/etc/githooks/scripts/python-code-pre-commit.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+exit 0
```
Comment (dehe1) [inline build/automation/etc/githooks/scripts/python-code-pre-commit.sh:4]: Is there a reason behind this?

Comment (terado) [reply build/automation/etc/githooks/scripts/python-code-pre-commit.sh:4]: This is what Dan added to ignore this step whilst it's non-functional

**Review by dehe1 — DISMISSED** [review]
LGTM, let's make a note to re-enable the checks as soon as they are fixed.

**Review by forjayssake — APPROVED** [review]
Re-approved after rebase

**Review by dehe1 — APPROVED** [review]

**Pull request merged**